front/src/modules/ui/components/buttons/Button.tsx

```diff
@@ -130,23 +147,47 @@ const StyledButton = styled.button<
   }
 `;
 
+const SoonPill = styled.span`
+  align-items: center;
+  background: ${({ theme }) => theme.background.transparent.light};
+  border-radius: 50px;
+  color: ${({ theme }) => theme.font.color.light};
+  display: flex;
+  font-family: Inter;
+  font-size: 11px;
+  font-style: normal;
+  font-weight: 500;
+  gap: 8px;
+  height: 16px;
+  justify-content: flex-end;
+  line-height: 150%;
+  margin-left: auto;
+  padding: 4px 8px;
+`;
+
 export function Button({
   icon,
   title,
   fullWidth = false,
   variant = 'primary',
   size = 'medium',
+  position = 'solo',
+  soon = false,
```

Comment:
@FelixMalfait Maybe we can just add an `accessory` props to  `Button` allowing to pass any component, as`Soon` pill already exist and it will avoid breaking `Button` api